components/scream/src/share/io/scream_scorpio_interface.hpp

```diff
@@ -55,10 +55,14 @@ namespace scorpio {
   /* Called each timestep to update the timesnap for the last written output. */
   void pio_update_time(const std::string &filename, const Real time);
 
-  /* Read data for a specific variable from a specific file. */
-  void grid_read_data_array (const std::string &filename, const std::string &varname, const int time_index, void* hbuf);
+  // Read data for a specific variable from a specific file. To read data that
```

Comment:
IIRC, putting an "invalid" time index was also a way to signal "read the last time index, if time is one of the dimensions". If so, perhaps we could make this clear in the comment.